### DISCUSSION TITLE
Documentation confusing for ADComputeVariableIsotropicElasticityTensor

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#18896] [Q&A General] [answered]
Hi All

Im working with a problem with lots of ADComputeVariableIsotropicElasticityTensor's, the documentation https://mooseframework.inl.gov/source/materials/ADComputeVariableIsotropicElasticityTensor.html states that the youngs_modulus entry is;
![Screenshot from 2021-09-22 18-55-36](https://user-images.githubusercontent.com/2439754/134396365-26032a30-f175-4401-b286-cc033b1c1e9e.png)
 Which to me reads that if I have the following setup
```
[Materials]
  [./fred_youngs_modulus]
    type = ADPiecewiseLinearInterpolationMaterial
    xy_data = '0          10e+6
               599.9999   10e+6
               600        9.94e+6
               99900      10e3'
    property = youngs_modulus
    variable = temp
  [../]

  [./elasticity_tensor]
    type = ADComputeVariableIsotropicElasticityTensor
    youngs_modulus = fred_youngs_modulus
    poissons_ratio = 0.0
  [../]
[]
```
Then all should be well, but it isnt, my material with name fred_youngs_modulus isn't found, I interpret the name to be the name of the HIT block. It isn't instead it needs to be the name of the property returned by the ADPieceWiseLinearInterpolation object. Is that a bug/intentional, either way I think the documentation should be a little clearer. Happy to submit a documentation PR if it helps.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Thank you for reporting this. I ll add this to a documentation fix PR I have open.
I ll look for other instances too.

Guillaume

**Reply by GiudGiud** [reply]
Fixed by #18866
